src/main/kotlin/graphics/scenery/backends/vulkan/VulkanObjectState.kt

```diff
@@ -44,6 +45,9 @@ open class VulkanObjectState : NodeMetadata {
     /** UBOs required by the [graphics.scenery.Node] this metadata object is attached to. */
     var UBOs = LinkedHashMap<String, Pair<Long, VulkanUBO>>()
 
+    //Like this here?
+    var SSBOs = LinkedHashMap<String, Pair<Long, VulkanUBO>>()
```

Comment:
The intent here is to have this work the same way as the UBO hash map, right? So you'd basically store the SSBO name, together with the Vulkan handle, and the actual UBO defining the data structure as value.

Reply:
Yes, i think that was my train of thought. Although Im not sure if we should refactor UBOS and SSBOs a bit. because Vulkan differentiates between ubos and ssbos, I needto extent the VulkanUBO class anyways. But notsure if this is 'trivially' doable while keepeing backwards-comp.